src/components/Map/index.tsx

```diff
@@ -39,22 +20,20 @@ import {
   ZINDEX_MARKER_MOUSE_OUT,
   ROUTE,
   SIDO_NAMES,
+  INIT_DUST_INFO,
 } from '@/utils/constants';
+
+import { makeCityMaker, makeSidoMaker, setMakerToNull } from '@/utils/makers';
```

Comment:
```suggestion
import { makeCityMarker, makeSidoMarker, setMakerToNull } from '@/utils/makeMarkers';
```
어떤 동작을 하는지 더 명확하면 좋을 것 같아요!

Reply:
makers 파일에 make(마커를 만듬)과 remove(마커를 지움)이 같이 들어있어서 markers로 했는데 makeMakers가 더 적절할까요? 고민이 되네요.